commit d3e0cfe0f12dcade20d5d1ea8bc51fee609bbc06
Author: Alice Dahan
AuthorDate: 2026-02-13 11:36:22 +0100
CommitDate: 2026-03-30 16:00:14 +0200

    chore: remplace la formule de la retraite de base par celle de l'Urssaf



diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/cotisations.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/cotisations.publicodes"
index 9c638420b..11ec66718 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/cotisations.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/cotisations-et-contributions/cotisations/cotisations.publicodes"
@@ -253,11 +253,20 @@ indépendant . cotisations et contributions . cotisations . retraite de base:
   variations:
     - si: profession libérale . réglementée . CNBF
       alors: profession libérale . réglementée . CNBF . retraite de base
+    - si: profession libérale . CNAVPL
+      alors: profession libérale . CNAVPL . retraite de base
     - sinon:
-        somme:
-          - tranche 1
-          - tranche 2
-        abattement: profession libérale . réglementée . CARMF . participation CPAM retraite
+        # Le code de la sécurité sociale décrit un calcul identique à celui des PLR
+        # (taux T1 sur la part de l'assiette < PASS + taux T2 sur toute l'assiette)
+        # mais l'Urssaf calcule différemment (taux T2 sur la part de l'assiette > PASS
+        # uniquement) et a insisté pour que l'on reprenne leur méthode.
+        barème:
+          assiette: assiette
+          multiplicateur: PSS proratisé
+          tranches:
+            - taux: taux T1
+              plafond: 1
+            - taux: taux T2
   abattement:
     le maximum de:
       - exonérations . Acre . exonération
@@ -274,20 +283,12 @@ indépendant . cotisations et contributions . cotisations . retraite de base:
     Cotisations minimales: https://www.secu-independants.fr/cotisations/calcul-des-cotisations/cotisations-minimales/
     Article D633-2 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000048838330
     Guide de l’assurance vieillesse des professions libérales (2025): https://www.cnavpl.fr/wp-content/uploads/2025/07/Guide-2025.pdf#page=23
-    Article D633-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904636/2024-07-07
-
-indépendant . cotisations et contributions . cotisations . retraite de base . tranche 1:
-  produit:
-    - assiette
-    - taux
-  arrondi: oui
-  unité: €/an
+    Article D633-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904636
 
   avec:
     assiette:
       valeur: assiette retraite et invalidité-décès
       plancher: assiette minimale . retraite
-      plafond: PSS proratisé
       abattement:
         applicable si: déduction tabac
         valeur: déduction tabac
@@ -296,52 +297,21 @@ indépendant . cotisations et contributions . cotisations . retraite de base . t
         Article D633-2 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000048838330
         Article D642-4 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000048838321
 
-    taux:
+    taux T1:
       variations:
-        - si: profession libérale . CNAVPL
-          alors:
-            variations:
-              - si: après réforme
-                alors: 8.73%
-              - sinon: 8.23%
-        - sinon: 17.15%
+        - si: après réforme
+          alors: 17.87%
+        - sinon: 17.75%
       références:
         Article D633-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904636
-        Article D642-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904651
-
-indépendant . cotisations et contributions . cotisations . retraite de base . tranche 2:
-  produit:
-    - assiette
-    - taux
-  arrondi: oui
-  unité: €/an
 
-  avec:
-    assiette:
-      valeur: assiette retraite et invalidité-décès
-      plancher: assiette minimale . retraite
-      plafond:
-        applicable si: profession libérale . CNAVPL
-        valeur: 5 * PSS proratisé
-      abattement:
-        applicable si: déduction tabac
-        valeur: déduction tabac
-      références:
-        Article D633-2 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000048838330
-        Article D642-4 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000048838321
-
-    taux:
+    taux T2:
       variations:
-        - si: profession libérale . CNAVPL
-          alors: 1.87%
-        - sinon:
-            variations:
-              - si: après réforme
-                alors: 0.72%
-              - sinon: 0.60%
+        - si: après réforme
+          alors: 0.72%
+        - sinon: 0.60%
       références:
         Article D633-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904636
-        Article D642-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904651
 
 indépendant . cotisations et contributions . cotisations . retraite complémentaire:
   description: |
diff --git "a/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNAVPL.publicodes" "b/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNAVPL.publicodes"
index 9c1d0c756..3eae362fe 100644
--- "a/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNAVPL.publicodes"
+++ "b/modele-ti/r\303\250gles/ind\303\251pendant/professions-lib\303\251rales/CNAVPL.publicodes"
@@ -43,6 +43,42 @@ indépendant . profession libérale . CNAVPL . maladie:
           - plafond: 110%
             taux: 6.5%
 
+indépendant . profession libérale . CNAVPL . retraite de base:
+  somme:
+    - tranche 1
+    - tranche 2
+  abattement: profession libérale . réglementée . CARMF . participation CPAM retraite
+  références:
+    Guide de l’assurance vieillesse des professions libérales (2025): https://www.cnavpl.fr/wp-content/uploads/2025/07/Guide-2025.pdf#page=23
+    Article D642-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000049904651
+
+  avec:
+    tranche 1:
+      produit:
+        - valeur: cotisations et contributions . cotisations . retraite de base . assiette
+          plafond: PSS proratisé
+        - taux
+      arrondi: oui
+      unité: €/an
+
+      avec:
+        taux:
+          variations:
+            - si: cotisations et contributions . après réforme
+              alors: 8.73%
+            - sinon: 8.23%
+
+    tranche 2:
+      produit:
+        - valeur: cotisations et contributions . cotisations . retraite de base . assiette
+          plafond: 5 * PSS proratisé
+        - 1.87%
+      arrondi: oui
+      unité: €/an
+
+      avec:
+        taux: 1.87%
+
 indépendant . profession libérale . CNAVPL . retraite complémentaire:
   variations:
     - si: profession libérale . réglementée . CARCDSF
diff --git "a/modele-ti/r\303\250gles/protection-sociale/retraite-base.publicodes" "b/modele-ti/r\303\250gles/protection-sociale/retraite-base.publicodes"
index 71d9a9d1e..5058a88d2 100644
--- "a/modele-ti/r\303\250gles/protection-sociale/retraite-base.publicodes"
+++ "b/modele-ti/r\303\250gles/protection-sociale/retraite-base.publicodes"
@@ -34,12 +34,11 @@ protection sociale . retraite . base:
   arrondi: oui
   unité: €/mois
   références:
-    'Retraites de base et complémentaire dans le privé : quelles différences ?': https://www.service-public.fr/particuliers/vosdroits/F12389
     Assurance Retraite de la Sécurité sociale: https://www.lassuranceretraite.fr/
-    Calcul de la retraite du salarié du secteur privé: https://www.service-public.fr/particuliers/vosdroits/F21552
-    Régime de retraite des travailleurs indépendants: https://entreprendre.service-public.fr/vosdroits/F33841
-    Montant de la retraite personnelle: https://legislation.lassuranceretraite.fr/#/expose?file_leaf_ref=retraite_personnelle_montant_montant_ex.aspx
+    Régime de retraite de l'entrepreneur individuel (EI): https://entreprendre.service-public.gouv.fr/vosdroits/F33841
     Durée d’assurance pour le taux plein: https://legislation.lassuranceretraite.fr/#/expose?file_leaf_ref=retraite_personnelle_taux_duree_assurance_pour_taux_plein_ex.aspx
+    Article L161-17-3 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000028494794/
+    Montant de la retraite personnelle: https://legislation.lassuranceretraite.fr/#/expose?file_leaf_ref=retraite_personnelle_montant_montant_ex.aspx
 
 protection sociale . retraite . base . trimestres:
   titre: trimestres validés
@@ -60,10 +59,8 @@ protection sociale . retraite . base . trimestres:
     - exonération invalidité indépendant
   plafond: 4 trimestre validé/an
   références:
-    Durée d’assurance retraite du salarié du secteur privé: https://www.service-public.fr/particuliers/vosdroits/F1761
-    # Lien mort :
-    # Salaire validant un trimestre: https://www.legislation.cnav.fr/#/bareme?file_leaf_ref=salaire_validant_un_trimestre_montant_bar.aspx
-    Article R351-9 du code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000028751530/2014-03-21
+    Article R351-9 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000028751530
+    Guide de l’assurance vieillesse des professions libérales (2025): https://www.cnavpl.fr/wp-content/uploads/2025/07/Guide-2025.pdf#page=26
 
   avec:
     exonération invalidité indépendant:
@@ -76,64 +73,34 @@ protection sociale . retraite . base . trimestres:
 
 protection sociale . retraite . base . revenu cotisé:
   titre: revenu cotisé pour la retraite de base
-  arrondi: oui
   valeur:
     indépendant . cotisations et contributions . cotisations . retraite de
-    base / indépendant . cotisations et contributions . cotisations . retraite
-    de base . tranche 1 . taux
+    base / taux de cotisation
   plafond: plafond sécurité sociale
+  arrondi: oui
   description: |
     Le revenu cotisé pour la retraite de base est égal à :
 
     [Cotisations à la retraite de base](/documentation/modele-ti/indépendant/
     cotisations-et-contributions/cotisations/retraite-de-base) / [taux de la
     cotisation à la retraite de base](/documentation/modele-ti/indépendant/
-    cotisations-et-contributions/cotisations/retraite-de-base/tranche-1/taux)
+    cotisations-et-contributions/cotisations/retraite-de-base/taux-T1)
 
     En cas d’exonération de cotisations, un montant plancher est utilisé pour le
     calcul.
 
     Dans tous les cas, le résultat est limité au plafond de la sécurité sociale.
   références:
-    Article R351-9 du code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000028751530
-    Article R634-1 du code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000038787378
-    Guide de l’assurance vieillesse des professions libérales (2025): https://www.cnavpl.fr/wp-content/uploads/2025/07/Guide-2025.pdf
-  note: |-
-    Rappel : la validation de trimestres s’effectue, pour les travailleurs
-    indépendants à partir du **revenu cotisé**
-
-    Le dernier alinéa de l’article R351-9  du code de la sécurité sociale, tel
-    que rendu applicable aux travailleurs indépendants par les articles D634-1 et
-    R634-1 du même code, moyennant adaptations, prévoit que pour la période
-    postérieure au 31 décembre 2013, il y a lieu de retenir autant de trimestres
-    que le revenu annuel correspondant aux retenues subies par l’assuré
-    représente de fois le montant du salaire minimum de croissance en vigueur au
-    1er janvier de l’année considérée calculé sur la base de 150 heures, avec un
-    maximum de quatre trimestres par année civile.
-
-    Par retenues subies par l’assuré, le texte vise (s’agissant d’un indépendant
-    qui n’est pas forcément à jour de ses cotisations) les cotisations du régime
-    vieillesse de base effectivement versées par lui (et non des cotisations
-    appelées).
-
-    Par revenu annuel correspondant aux retenues subies par l’assuré (dit
-    « revenu cotisé ») on entend le revenu reconstitué à partir des cotisations
-    du régime vieillesse de base (« RVB ») effectivement payées par l’assuré.
-
-    Le revenu cotisé correspond donc à la cotisation du régime vieillesse de
-    base effectivement versée au titre de l’année N divisée par le taux de
-    cotisation du régime vieillesse de base de cette année N.
-
-    Cotisation définitive RVB effectivement versée au titre d’une année N
-    (+réductions ou exonérations génératrices de droits)  / taux de cotisation
-    RVB au titre de cette année N (pour la part génératrice de droit applicable
-    au revenu plafonné).
-
-    Ce revenu (dit « revenu cotisé » d’une année N) est retenu pour la
-    validation de trimestres d’une année N.
-
-    Il est comparé au seuil de 150 smic/h pour déterminer le nombre de trimestres
-    à valider (revenu cotisé de l’année N /150 smic h).
+    Article R351-9 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000028751530
+    Article R634-1 du Code de la sécurité sociale: https://www.legifrance.gouv.fr/codes/article_lc/LEGIARTI000038787378
+
+  avec:
+    taux de cotisation:
+      variations:
+        - si: indépendant . profession libérale . CNAVPL
+          alors: indépendant . profession libérale . CNAVPL . retraite de base . tranche 1 . taux
+        - sinon: indépendant . cotisations et contributions . cotisations . retraite
+            de base . taux T1
 
 protection sociale . retraite . base . CNAVPL:
   applicable si: indépendant . profession libérale . CNAVPL
@@ -145,10 +112,10 @@ protection sociale . retraite . base . CNAVPL:
     Le nombre de points acquis chaque année est déterminé en fonction des revenus
     professionnels soumis à cotisations :
 
-    - la cotisation maximale sur la tranche T1 permet d’acquérir 525 points (de 0 à
-    un plafond annuel de la sécurité sociale),
+    - la cotisation maximale sur la tranche T1 permet d’acquérir 557 points (de 0 à
+    1 plafond annuel de la sécurité sociale),
     - la cotisation maximale sur la tranche T2 permet d’acquérir 25 points (de 0 à
-    cinq fois le plafond annuel de la sécurité sociale).
+    5 fois le plafond annuel de la sécurité sociale).
 
     Pour chaque tranche, le nombre de points attribué est calculé au prorata de la
     cotisation effective sur la cotisation maximale et arrondi à la décimale la plus
@@ -160,7 +127,7 @@ protection sociale . retraite . base . CNAVPL:
   unité: €/mois
   références:
     Comprendre sa retraite - Pension du professionnel libéral - Montant de la pension: https://www.cnavpl.fr/comprendre-sa-retraite/
-    Guide de l’assurance vieillesse des professions libérales: https://www.cnavpl.fr/statuts-et-documents-de-reference/?wpdmdl=234211#page=27
+    Guide de l’assurance vieillesse des professions libérales (2025): https://www.cnavpl.fr/wp-content/uploads/2025/07/Guide-2025.pdf#page=27
 
   avec:
     valeur du point:
@@ -196,26 +163,26 @@ protection sociale . retraite . base . CNAVPL . points acquis:
             - si: indépendant . cotisations et contributions . après réforme
               alors: 557 point
             - sinon: 525 point
-        - indépendant . cotisations et contributions . cotisations . retraite de base . tranche 1 /
+        - indépendant . profession libérale . CNAVPL . retraite de base . tranche 1 /
           cotisation maximum T1
       arrondi: 1 décimale
 
     cotisation maximum T1:
       privé: oui
-      valeur: indépendant . cotisations et contributions . cotisations . retraite de base . tranche 1
+      valeur: indépendant . profession libérale . CNAVPL . retraite de base . tranche 1
       contexte:
         indépendant . cotisations et contributions . assiette sociale: plafond sécurité sociale
 
     points T2:
       produit:
         - 25 point
-        - indépendant . cotisations et contributions . cotisations . retraite de base . tranche 2 /
+        - indépendant . profession libérale . CNAVPL . retraite de base . tranche 2 /
           cotisation maximum T2
       arrondi: 1 décimale
 
     cotisation maximum T2:
       privé: oui
-      valeur: indépendant . cotisations et contributions . cotisations . retraite de base . tranche 2
+      valeur: indépendant . profession libérale . CNAVPL . retraite de base . tranche 2
       contexte:
         indépendant . cotisations et contributions . assiette sociale: 5 * plafond sécurité sociale
 
